src/Controller/ListCartsController.php

```diff
@@ -14,10 +15,15 @@
  */
 class ListCartsController
 {
-    public function __invoke(Environment $twig, CartRepository $cartRepository)
+    public function __invoke(Request $request, Environment $twig, CartRepository $cartRepository)
     {
+        $label = $request->get('state');
```

Comment:
should have used `$request->request->get('state')` or `$request->query->get('state')` instead.